### PR TITLE
Completed table and partial

### PR DESCRIPTION
**Pull request by hangloos**
-Added contact info partial
-Added mobile tag to application.html.erb
- Fixed mobile text and table issues

### CODE REVIEW

**Comment by hangloos** [comment]
https://trello.com/c/P1ptX6MC/107-make-a-partial-for-contact-info

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -12,4 +12,5 @@
  *
  *= require_tree .
  *= require_self
+ *= require _colors
```
Comment (gatorjuice) [inline app/assets/stylesheets/application.css]: This is already being pulled in without being referenced here

File: app/assets/stylesheets/pages.scss [hunk 2]
```diff
@@ -3,5 +3,19 @@
...
   text-align: center;
+  }
+
+@media only screen and (max-width: 768px) {
+    .contact-us {
+        font-size: 1.25em;
+    }
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (gatorjuice) [inline app/assets/stylesheets/pages.scss]: these can be grouped together in sass

File: app/assets/stylesheets/quotes.scss [hunk 3]
```diff
@@ -3,3 +3,8 @@
 .p {
   color: $wet-concrete
 }
+
```
Comment (gatorjuice) [inline app/assets/stylesheets/quotes.scss]: extra line

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>Veterans Work</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=0">
```
Comment (gatorjuice) [inline app/views/layouts/application.html.erb]: indentation

File: app/views/pages/_contact_info.html.erb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+ <h2>Contact Us:</h2>
+    <div class="row">
```
Comment (gatorjuice) [inline app/views/pages/_contact_info.html.erb]: indentation

File: app/views/pages/about.html.erb [hunk 6]
```diff
@@ -59,12 +59,7 @@
...
 <a class="navbar-brand" href="/"><%= image_tag "veterans_work_logo_no_words.png", height: "40", alt: "Company Logo" %></a>
 
-<h2>Contact Us:</h2>
-  <div class="row">
-    <div class="well">
-      <h3><u>Email</u>: veteransworkheroku@gmail.com</h3>
-      <h3><u>Phone</u>: 123-456-7890</h3>
-    </div>
-  </div>
+  <%= render 'contact_info' %>
```
Comment (gatorjuice) [inline app/views/pages/about.html.erb]: indendation

Comment (gatorjuice) [inline app/views/pages/about.html.erb]: the render statement is fine, its the three lines above it

**Pull request closed without merging**